turbo.json

```diff
@@ -3,14 +3,8 @@
 	"remoteCache": {
 		"signature": true
 	},
-	"globalEnv": [
-		"CI_OS",
-		"NODE_VERSION",
-		"VITEST",
-		"NODE_DEBUG",
-		"NODE_EXTRA_CA_CERTS",
-		"MINIFLARE_WORKERD_PATH"
-	],
+	"globalEnv": ["CI_OS", "NODE_VERSION"],
```

Comment:
Let's ensure that we cache separately for each OS and Node.js version.
Particularly relevant for our shared CI cache.